services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -69,7 +70,7 @@ async def _abort_pipeline_tasks(
 
 @router.post(
     "",
-    summary="Create and Start a new computation",
+    summary="Create and optionally start a new computation",
```

Comment:
I wonder if it would make sense to have ``create`` and ``start`` options separated as in the docker containers interface, in which ``run = create + start``. I do not mean there is a one to one equivalence though.

In any case, we need to unify a bit more the language. I find it confusing e.g. how you associate ``task.start_pipeline`` with ``publish`` running-state.

Reply:
I was wondering also... will keep it for another time